app/viewModels/exit/RemoveOfficeOfExitViewModel.scala

```diff
@@ -29,14 +29,11 @@ case class RemoveOfficeOfExitViewModel(officeOfExit: Option[CustomsOffice]) {
 
   def heading(implicit messages: Messages): String = messages(s"$prefix.heading", args: _*)
 
-  val prefix: String = {
-    val key = "exit.index.confirmRemoveOfficeOfExit"
-    officeOfExit.fold(s"$key.default")(
-      _ => key
-    )
-  }
+  val prefix: String = "exit.index.confirmRemoveOfficeOfExit"
 
   val args: Seq[String] = officeOfExit.map(_.name).toList
+
+  val officeName: String = args.headOption.getOrElse("")
```

Comment:
Rather than having an empty inset text if the string is empty, it might be preferable to have this be an Option[String], and only render the inset text if the option is defined. Could then refactor to be

```
val officeName: Option[String] = officeOfExit.map(_.name)
val args = officeName.toList
```

Reply:
Actually I assume `args` is no longer used if we're not passing it into the title/h1 ?